scripts/runNpmAudit.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+for i in {1..5}
+do
+    npm audit
+    EXIT_CODE=$?
+
+    if [ $EXIT_CODE -eq 0 ]
+    then
+        # npm audit returned succesfully, exiting
+        exit 0
+    else
+        echo 'NPM audit failed, retrying...'
+    fi
+done
+
+exit 0
```

Comment:
As this always exits 0 does it not mean it could fail 5 times and then be successful anyway making the audit check obsolete on a genuine failure? 

I'd assume we'd want an if statement checking `i` isn't `5`

Reply:
it could yes......... How about we just change the default exit code to 1 instead of 0 so it it does fail 5 times it will exit with a figure by default, prompting a dev to investigate?

Reply:
Yeah we should fail if we haven't returned successfully within the 5 tries